app/frontends/workflow-ui.ts

```diff
@@ -128,13 +139,18 @@ function jobRenderingFunctions(logger: Logger, requestQuery: Record<string, any>
       try {
         const url = new URL(this.request);
         const path = url.pathname + url.search;
-        return path;
+        return truncateString(path, 100);
```

Comment:
Maybe move these hardcode lengths into a constant or env var to make it easy to change them all at once if needed.